mmv1/third_party/terraform/services/bigquery/resource_bigquery_table.go.tmpl

```diff
@@ -992,6 +1009,25 @@ func ResourceBigQueryTable() *schema.Resource {
 				DiffSuppressFunc: bigQueryTableSchemaDiffSuppress,
 				Description:      `A JSON schema for the table.`,
 			},
+			{{- if ne $.TargetVersionName "ga" }}
+			// SchemaForeignTypeInfo: [Optional] Specifies metadata of the foreign data type definition in field schema.
+			"schema_foreign_type_info": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    1,
+				Description: "Specifies metadata of the foreign data type definition in field schema.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						// TypeSystem: [Optional] Specifies the system which defines the foreign data type.
+						"type_system": {
+							Type:             schema.TypeString,
+							Optional:         true,
```

Comment:
Can we make this field required, given this is currently the only subfield?

Reply:
Thanks for catching this, it should be required.